backend/src/federation/index.ts

```diff
@@ -28,7 +41,9 @@ export const startDHT = async (
   try {
     const TOPIC = DHT.hash(Buffer.from(topic))
 
-    const keyPair = DHT.keyPair()
+    const keyPair = DHT.keyPair(getSeed())
+    logger.info(`keyPairDHT: publicKey=${keyPair.publicKey.toString('hex')}`)
+    logger.info(`keyPairDHT: secretKey=${keyPair.secretKey.toString('hex')}`)
```

Comment:
do we want to log the secret key?

Reply:
Not necessary at all.
both keys are logged to see differences between with and without seed creation.
shift the secret-log to debug.level